pkg/authorization/workspace_content_authorizer.go

```diff
@@ -129,12 +168,28 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 	ws, err := a.clusterWorkspaceLister.Get(clusters.ToClusterAwareKey(parentClusterName, cluster.Name.Base()))
 	if err != nil {
 		if errors.IsNotFound(err) {
+			kaudit.AddAuditAnnotations(
+				ctx,
+				WorkspaceContentAuditDecision, DecisionDenied,
+				WorkspaceContentAuditReason, "clusterworkspace not found",
```

Comment:
fmt.Sprintf(`parent clusterworkspace %q not found`, parentClusterName)

Reply:
this is checking for the workspace itself, not the parent

Reply:
ah yes, the parent was the prefix only in get, my bad.